### PR TITLE
Get user info

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Get user infomation
## Reference Requirements
- Task ID: #12

## Reference Isuees
#{issue_id}

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)
```
mysql> Explain select * from `users` where `users`.`id` = '2' and `users`.`deleted_at` is null limit 1\G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: users
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```

### CODE REVIEW

**Comment by at-haphan** [comment]
LGTM

**Comment by at-haytran** [comment]
Update code

File: app/Http/Controllers/Admin/UserController.php [hunk 1]
```diff
@@ -24,11 +24,14 @@ public function index()
...
      *
+     * @param int $id id of user
+     *
      * @return view
      */
-    public function edit()
+    public function edit($id)
     {
-        return view('admin.users.update');
+        $users = User::find($id);
```
Comment (at-huydq) [inline app/Http/Controllers/Admin/UserController.php]: dùng findOrFail giúp a nhé.

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**